puppet/modules/web/manifests/vhost/web.pp

```diff
@@ -26,12 +26,13 @@
     { 'rewrite_rule' => ['^/wiki(.*)  https://projects.theforeman.org/wiki$1  [R,L]'] },
     { 'rewrite_rule' => ['^/events/all.ics https://community.theforeman.org/c/events/l/calendar.ics [R,L]'] },
     { 'rewrite_rule' => ['^/events/? https://community.theforeman.org/c/events/l/calendar [R,L]'] },
+    { 'rewrite_rule' => ["^/api/latest(.*) https://apidocs.theforeman.org/foreman/${stable}\$1 [R,L]"] },
+    { 'rewrite_rule' => ["^/api/(.*) https://apidocs.theforeman.org/foreman/\$1 [R,L]"] },
```

Comment:
This won't capture `https://theforeman.org/api` and if we remove the directory, I think there won't be a redirect to the trailing slash leading to a 404 for users.

Reply:
But this is all I can do, I cannot want to redirect everything starting with "api". We actually do have an old page named `api.html`: https://theforeman.org/api.html and that would mask it.

Reply:
One easy solution:
```suggestion
    { 'rewrite_rule' => ["^/api/(.*) https://apidocs.theforeman.org/foreman/\$1 [R,L]"] },
    { 'rewrite_rule' => ["^/api$ https://apidocs.theforeman.org/foreman/ [R,L]"] },
```

You'll also see that there's a single regex line as well but I'll leave that as an excise to the reader :)

Reply:
Sure I can do that, amended.